### DISCUSSION TITLE
How To: Setup Cascaded Editors

### DISCUSSION THREAD

**Discussion by harifenny** [serenity-is/Serenity#5370] [Help] [answered]
I just want to try setup cascaded editors with this link

[How To: Setup Cascaded Editors](https://volkanceylan.gitbooks.io/serenity-guide/content/howto/how_to_setup_cascaded_editors.html)

But I don't know how to add this code

`this.Items.Where(x => x.CountryId == CountryEditorValue)`

How to add this code? And where is to add file?

Thanks a lot, 
Hari Setyawan

**Comment by VictorTomaili** [comment, marked as answer]
You don't need write this code, it's works automatically in background. Just sample for imagine how it's filter.

**Reply by harifenny** [reply]
Thanks a lot Victor

**Comment by erehtolleh** [comment]
@harifenny  Victor is right, that code is just the explanation of how the editor filters but you don't need to add that.  All the sample above this line of code should gets you going with the cascaded editors.

**Reply by harifenny** [reply]
Thank you
